packages/preview/mahjong-tiles/0.1.0/typst.toml

```diff
@@ -0,0 +1,14 @@
+[package]
+name = "mahjong-tiles"
```

Comment:
This name is too descriptive of what the package does. Could you please change it to something more unique?